### ISSUE TITLE
360  Capture rendering to picture

### ISSUE THREAD

**Issue by qq294716498** [RenderHeads/UnityPlugin-AVProMovieCapture#67]
Sometimes,  only need to take a screenshot, and the rendering sequence frame will render many

**Comment by qq294716498** [comment]
Rendering sequence frames, the name of PNG cannot be customized

**Comment by qq294716498** [comment]
I use windows, unity2019.3.3hdrp. After rendering, my project will be very slow and will have to be restarted

**Comment by AndrewRH** [comment]
> Rendering sequence frames, the name of PNG cannot be customized

Yes it can be.  If you're using the component you can set the filename here:
![image](https://user-images.githubusercontent.com/5418126/89890223-e62ff200-dbca-11ea-963b-d057a889681a.png)

If you're using the tool window, then you can set the file name here:

![image](https://user-images.githubusercontent.com/5418126/89890283-fd6edf80-dbca-11ea-957c-f59a34190a3c.png)

**Comment by AndrewRH** [comment]
> Sometimes, only need to take a screenshot, and the rendering sequence frame will render many

You can set the number of frames here:
![image](https://user-images.githubusercontent.com/5418126/89890363-2000f880-dbcb-11ea-9dc4-57f47a50b19d.png)

In the future though we do plan to add a snapshot mode that will just save a single image.

**Comment by AndrewRH** [comment]
> I use windows, unity2019.3.3hdrp. After rendering, my project will be very slow and will have to be restarted

Thanks for reporting this - could you tell us how you're using the plugin to achieve this effect?  Is it possible to replicate it using one of our demo scenes?

How exactly is the project slow, and which platform is this on?

Thanks,
